### ISSUE TITLE
The loading bar should be in front of the text in summary page

### ISSUE THREAD

**Issue by mathewleeza** [microsoft/devhome#499]
### Steps to reproduce the bug

Description
The loading ring should be in front of the text rather than behind the text in current set up setting up page .
